### PR TITLE
PILOT-1527: Add total_per_zone calculation in response for project-files statistics endpoint

### PR DESCRIPTION
**Pull request by afmam**
## Summary

- Added `total_per_zone` calculation for project-file statistics endpoint, which is dependent on user role (platform-admin, collaborator, contributor)

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1527

## Type of Change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Testing

Are there any new or updated tests to validate the changes?

- [X] Yes
- [ ] No

## Test Directions

- The number of files displayed for greenroom and core under "Go to" tab in Pilot should accurately reflect project roles (platform-admin, collaborator, contributor).

### CODE REVIEW

File: api/api_project_files.py [hunk 1]
```diff
@@ -63,26 +64,73 @@ def _replace_zone_labels_in_search_response(response: Dict[str, Any]) -> Dict[st
...
+            project_code, {**params, **{'zone': get_zone_int(ConfigClass.GREENROOM_ZONE_LABEL)}}
+        )
+        result_core = await search_service_client.get_project_statistics(
+            project_code, {**params, **{'zone': get_zone_int(ConfigClass.CORE_ZONE_LABEL)}}
+        )
+        stats_result = _add_file_stats_per_zone(
+            {ConfigClass.GREENROOM_ZONE_LABEL: result_greenroom, ConfigClass.CORE_ZONE_LABEL: result_core}
+        )
+
+    if project_role == 'collaborator':
```
Comment (colorzzr) [inline api/api_project_files.py:113]: small update: I guess here can be `elif` to make a clearer logic

File: api/api_project_files.py [hunk 2]
```diff
@@ -63,26 +64,73 @@ def _replace_zone_labels_in_search_response(response: Dict[str, Any]) -> Dict[st
...
+        )
+        result_core = await search_service_client.get_project_statistics(
+            project_code, {**params, **{'zone': get_zone_int(ConfigClass.CORE_ZONE_LABEL)}}
+        )
+
+        stats_result = _add_file_stats_per_zone(
+            {ConfigClass.GREENROOM_ZONE_LABEL: result_greenroom, ConfigClass.CORE_ZONE_LABEL: result_core}
+        )
+
+    if project_role == 'contributor':
```
Comment (colorzzr) [inline api/api_project_files.py:126]: small update: I guess here can be `elif` to make a clearer logic

**Review by colorzzr — APPROVED** [review]

File: api/api_project_files.py (hunk 1 above)
Comment (afmam) [reply api/api_project_files.py]: Fixed this

File: api/api_project_files.py (hunk 2 above)
Comment (afmam) [reply api/api_project_files.py]: Fixed this

**Pull request merged**